### ISSUE TITLE
vsk run examples/quick_draw gives TypeError: 'NoneType' object is not callable

### ISSUE THREAD

**Issue by mansi-aggarwal-2504** [abey79/vsketch#118]
I am following all steps but getting this error when I run `vsk run examples/quick_draw`:

```
Traceback (most recent call last):
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/bin/vsk", line 6, in <module>
    from vsketch_cli.cli import cli
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch_cli/cli.py", line 11, in <module>
    import vsketch
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch/__init__.py", line 3, in <module>
    from .sketch_class import Param, ParamType, SketchClass
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch/sketch_class.py", line 10, in <module>
    from .vsketch import Vsketch
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch/vsketch.py", line 22, in <module>
    import vpype_cli
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vpype_cli/__init__.py", line 5, in <module>
    from .blocks import *
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vpype_cli/blocks.py", line 7, in <module>
    from .cli import BlockProcessor, cli, execute_processors
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vpype_cli/cli.py", line 126, in <module>
    @cli.result_callback()
TypeError: 'NoneType' object is not callable
(genart-venv) (base) invlabs-MacBook-Pro:vsketch invlab$ vsk run examples/quick_drawvsk run examples/quick_draw
Traceback (most recent call last):
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/bin/vsk", line 6, in <module>
    from vsketch_cli.cli import cli
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch_cli/cli.py", line 11, in <module>
    import vsketch
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch/__init__.py", line 3, in <module>
    from .sketch_class import Param, ParamType, SketchClass
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch/sketch_class.py", line 10, in <module>
    from .vsketch import Vsketch
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch/vsketch.py", line 22, in <module>
    import vpype_cli
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vpype_cli/__init__.py", line 5, in <module>
    from .blocks import *
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vpype_cli/blocks.py", line 7, in <module>
    from .cli import BlockProcessor, cli, execute_processors
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vpype_cli/cli.py", line 126, in <module>
    @cli.result_callback()
TypeError: 'NoneType' object is not callable
(genart-venv) (base) invlabs-MacBook-Pro:vsketch invlab$ vsk run examples/quick_draw                           
Traceback (most recent call last):
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/bin/vsk", line 6, in <module>
    from vsketch_cli.cli import cli
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch_cli/cli.py", line 11, in <module>
    import vsketch
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch/__init__.py", line 3, in <module>
    from .sketch_class import Param, ParamType, SketchClass
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch/sketch_class.py", line 10, in <module>
    from .vsketch import Vsketch
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vsketch/vsketch.py", line 22, in <module>
    import vpype_cli
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vpype_cli/__init__.py", line 5, in <module>
    from .blocks import *
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vpype_cli/blocks.py", line 7, in <module>
    from .cli import BlockProcessor, cli, execute_processors
  File "/Users/invlab/Desktop/ML/Generative-Art/vsketch/genart-venv/lib/python3.7/site-packages/vpype_cli/cli.py", line 126, in <module>
    @cli.result_callback()
TypeError: 'NoneType' object is not callable
```

**Comment by abey79** [comment]
try `pip install -U click`

**Comment by mansi-aggarwal-2504** [comment]
I am now getting this error: 
`ModuleNotFoundError: No module named 'click._bashcomplete'`

Probably because there was an error while `pip install -U click`:
```
ERROR: vpype 1.7.0a0 has requirement setuptools<52.0.0,>=51.0.0, but you'll have setuptools 41.2.0 which is incompatible.
ERROR: typer 0.3.2 has requirement click<7.2.0,>=7.1.1, but you'll have click 8.0.1 which is incompatible.
```

Guess there are internal compatibility issues?

**Comment by abey79** [comment]
It looks like typer is not yet compatible with click 8 (tiangolo/typer#278). Hopefully they will address this rapidly. If not, I'll have to remove typer in favour of click.

In the mean time, you may fix the issue by forcing vpype to 1.6.0:

```
pip install -U vpype==1.6.0
```

**Comment by mansi-aggarwal-2504** [comment]
I did that and the errors have gone. But when I run my project, and I can only see a python process running but the window doesn't open. What might be the cause for this?

<img width="195" alt="Screenshot 2021-06-03 at 7 19 26 PM" src="https://user-images.githubusercontent.com/60866597/120655758-9e9f8d00-c4a0-11eb-9963-ec4dd415b154.png">


<img width="1626" alt="Screenshot 2021-06-03 at 7 18 33 PM" src="https://user-images.githubusercontent.com/60866597/120655639-83348200-c4a0-11eb-8a36-1278038a4eb6.png">

**Comment by abey79** [comment]
You may check if pip install -U git+https://github.com/abey79/vsketch@fix-no-typer#egg=vsketch fixes the issue.

**Comment by mansi-aggarwal-2504** [comment]
It didn't work. Same issue

**Comment by tiangolo** [comment]
I just released Typer `0.4.0`, which is compatible with **both** Click 7 and Click 8. :tada:
